src/mail/mail.controller.ts

```diff
@@ -74,16 +73,4 @@ export class MailController {
       payload.evaluationSetId
     );
   }
-
```

Comment:
It looks like this endpoint is invoked by easey-quartz-scheduler/admin/Jobs/EmailQueueJobs/SubmissionReminderQueue.cs and easey-quartz-scheduler/admin/Jobs/EmailQueueJobs/SubmissionWindowQueue.cs.  

1. Are these usages valid? If so, how should this handled?
2. Are there other references to this endpoint?

Reply:
The call of this would discarded in the and correct logic would be applied with the work done in the ticket #5826